db.scheme

```diff
@@ -5,7 +5,8 @@ CREATE TABLE IF NOT EXISTS `person` (
 
 CREATE TABLE IF NOT EXISTS `restaurant` (
  `id`   INTEGER NOT NULL PRIMARY KEY AUTOINCREMENT,
- `name` TEXT NOT NULL
+ `name` TEXT NOT NULL,
+ `comment` TEXT NOT NULL
```

Comment:
Well we have a existing db which I don't want to manually re-import or something. especially as this is going to change a few more times.
Migration is not necessarily about versioning (though that isn't bad for itself either). Migration commands can be added for now for checking if the column doesn't exists then altering the table. I agree this isn't the point to think about backwards compatibility, so they can be temporary until we have a stable structure.